src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/mailing/MailingPageRequestHandler.java

```diff
@@ -1,54 +1,41 @@
 package pl.tomaszdziurko.jvm_bloggers.view.admin.mailing;
 
 
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import pl.tomaszdziurko.jvm_bloggers.mailing.BlogSummaryMailGenerator;
 import pl.tomaszdziurko.jvm_bloggers.mailing.IssueNumberRetriever;
 import pl.tomaszdziurko.jvm_bloggers.mailing.MailSender;
-import pl.tomaszdziurko.jvm_bloggers.settings.Setting;
-import pl.tomaszdziurko.jvm_bloggers.settings.SettingKeys;
-import pl.tomaszdziurko.jvm_bloggers.settings.SettingRepository;
+import pl.tomaszdziurko.jvm_bloggers.metadata.Metadata;
+import pl.tomaszdziurko.jvm_bloggers.metadata.MetadataKeys;
+import pl.tomaszdziurko.jvm_bloggers.metadata.MetadataRepository;
 import pl.tomaszdziurko.jvm_bloggers.utils.DateTimeUtilities;
 import pl.tomaszdziurko.jvm_bloggers.utils.NowProvider;
 
 @Component
+@NoArgsConstructor
+@AllArgsConstructor(onConstructor = @_(@Autowired))
```

Comment:
Should be `@__` (two underscores). Having one underscore generates compiler warning:
```
'_' should not be used as an identifier, since it is a reserved keyword from source level 1.8 on
```